frontend/src/static/index.js

```diff
@@ -1,2 +1,4 @@
 Elm = require('../elm/main');
 Elm.Main.embed(document.getElementById('main'));
+
+fileRequest = require('../js/filerequest');
```

Comment:
Bundling `filerequest.js` into `main.js`.
I'm not sure whether it is properly handled.